synapse/logging/opentracing.py

```diff
@@ -474,7 +474,11 @@ def init_tracer(hs: "HomeServer") -> None:
     # off point into the logs. Can also be used to filter for an
     # instance that is under load.
     tags[SynapseTags.INSTANCE_NAME] = hs.get_instance_name()
-
+    # adding host and port of jaeger if it's hosted on different machine
+    jaeger_config['local_agent'] = {
+        'reporting_host': hs.config.tracing.host,
+        'reporting_port': hs.config.tracing.port
+    }
```

Comment:
I think this is equivalent to putting the following in your homeserver config:

```yaml
opentracing:
  jaeger_config:
    local_agent:
      reporting_host: other-host.local
      reporting_port: 1234
```

Does that work? See the [config docs](https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html#opentracing-1).

Reply:
Yes! Thanks for sharing. Didn’t see it unfortunately. You can close the pull request 👍🏻